SimpleJavaProject/src/main/java/ana_raetcaia/NumbersMain.java

```diff
@@ -0,0 +1,42 @@
+package ana_raetcaia;
+
+import java.util.Scanner;
+
+public class NumbersMain {
+    public static void main (String[]args){
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter a number: ");
+        double number = scanner.nextInt();
+        System.out.println(Math.sqrt(number));
+
+        double num1 = Math.random();
+        int rezult1 = (int)(num1 *100+0);
+        System.out.println("Random number between 0 and 100:" + rezult1);
+
+        double num2 = Math.random();
+        int rezult2 = (int)(num2 *100+10);
+        System.out.println("Random number between 10 and 100:" + rezult2);
+
+
+        int numberOne = 4;
+        int numberTwo = 6;
+        int numberThree = 8;
+        int numberFour = 2;
+
+        int result1 = Math.max(numberOne,numberTwo);
+        int result2 = Math.max(result1,numberThree);
+        int result3 = Math.max(result2 ,numberFour);
+        System.out.println("Maximum of 4 numbers: " + result3);
+
+        NumbersMain AverageObj = new NumbersMain();
+        System.out.println("Average of three numbers:" + AverageObj.AverageOfThree(2,4,7));
+    }
+     Number AverageOfThree(int number1,int number2, int number3){
```

Comment:
Oh, are you sure what is happening here?
can you explain ?)

Reply:
I learned early about methods and objects.
So, I created a method that calculates the average of three numbers, after that, I created an object of class NumersMain and called a method through it.

Reply:
Fixed